File: 2-Java/1-Programas/2-EstructurasControl/1-Actividad De Desarrollo 1. Ejercicios de Estructuras de Control Selectivas/Ejercicio1.java
import javax.swing.JOptionPane;

public class Ejercicio1 {
    public static void main(String[] args) {

        double parcial1 = Double.parseDouble
        (JOptionPane.showInputDialog("Ingrese calificación del Primer Parcial:"));
        double parcial2 = Double.parseDouble
        (JOptionPane.showInputDialog("Ingrese calificación del Segundo Parcial:"));
        double proyecto = Double.parseDouble
        (JOptionPane.showInputDialog("Ingrese calificación del Proyecto Final:"));

        double promedio = (parcial1 + parcial2) / 2; 

        String estatus = "";
        if (promedio >= 70 && proyecto >= 60) {
            estatus = "Aprobado";
        } else if (promedio >= 50 && promedio <= 69) {
            estatus = "Extraordinario";
        } else {
            estatus = "Reprobado";
        }
        String mensajeExtra = "";

        if (promedio >= 95) {
            mensajeExtra = "\n¡Alumno con excelencia académica!";
        }
        JOptionPane.showMessageDialog(null, 
            "Promedio Final: " + promedio + 
            "\nEstatus: " + estatus + 
            mensajeExtra);
    }
}

File: 3-Python/1-Programas/1-EstructurasSimples/ejercicio1.py
parcial1 = float(input("Calificación Primer Parcial: "))
parcial2 = float(input("Calificación Segundo Parcial: "))
proyecto = float(input("Calificación Proyecto Final: "))

promedio = (parcial1 + parcial2) / 2

if promedio >= 70 and proyecto >= 60:
    estatus = "Aprobado"
elif 50 <= promedio <= 69:
    estatus = "Extraordinario"
else:
    estatus = "Reprobado"

mensaje_extra = ""
if promedio >= 95:
    mensaje_extra = "Alumno con excelencia académica"

print(f"--- Resultados ---\nPromedio Final: {promedio}")
print(f"Estatus: {estatus}")
if mensaje_extra:
    print(mensaje_extra)